### ISSUE TITLE
Valve will update VAC System, will Charlatano still work? 

### ISSUE THREAD

**Issue by lestercuizon** [Jire/Charlatano#947]
Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.

**Comment by Myth80** [comment]
> Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.

I tried it in the beta version, it works just fine.

**Comment by ThatGuyFromCSGO** [comment]
> > Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.
> 
> I tried it in the beta version, it works just fibut

> > Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.
> 
> I tried it in the beta version, it works just fine.

I'm afraid on getting banned since i use it on my main. Does it put your account at risk?

**Comment by karmalegend** [comment]
> > > Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.
> > 
> > 
> > I tried it in the beta version, it works just fibut
> 
> > > Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.
> > 
> > 
> > I tried it in the beta version, it works just fine.
> 
> I'm afraid on getting banned since i use it on my main. Does it put your account at risk?

Cheating by definition puts your account at risk.

**Comment by primenumbers1** [comment]
> > > Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.
> > 
> > 
> > I tried it in the beta version, it works just fibut
> 
> > > Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.
> > 
> > 
> > I tried it in the beta version, it works just fine.
> 
> I'm afraid on getting banned since i use it on my main. Does it put your account at risk?

Yes but at the same time no. It's yes if you're using it on your main, you'll level your profile up less and no, if you actually know how to hide them well

**Comment by karmalegend** [comment]
> > > > Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.
> > > 
> > > 
> > > I tried it in the beta version, it works just fibut
> > 
> > 
> > > > Recently, CS released a beta version which has a "Secure Mode" in it which blocks files incompatible with CSGO. It also blocks Nvidia, Discord and such. When Secure Mode is turned off, it impacts your trust factor and more likely to play with Rage Hackers.
> > > 
> > > 
> > > I tried it in the beta version, it works just fine.
> > 
> > 
> > I'm afraid on getting banned since i use it on my main. Does it put your account at risk?
> 
> Yes but at the same time no. It's yes if you're using it on your main, you'll level your profile up less and no, if you actually know how to hide them well

This is like saying stealing in a store won't matter if you come by a lot. Regardless you're doing something that's entitled to punishment. It doens't matter in what state.
